README.md

```diff
@@ -46,7 +46,8 @@ Payment methods can accept preferences either directly entered in admin, or from
         environment: Rails.env.production? ? 'production' : 'sandbox',
         merchant_id: ENV['BRAINTREE_MERCHANT_ID'],
         public_key: ENV['BRAINTREE_PUBLIC_KEY'],
-        private_key: ENV['BRAINTREE_PRIVATE_KEY']
+        private_key: ENV['BRAINTREE_PRIVATE_KEY'],
+        paypal_flow: 'vault', # 'checkout' is accepted too
```

Comment:
isn't `checkout_with_paypal` in the comment?

Reply:
@kennyadsl updated.